docs/SOPs/dataset_acknowledgement_SOP.md

```diff
@@ -34,48 +34,40 @@ At any point before halfway through the sprint, the wrangler/person responsible
 #### Requirements
 
 - Python3
-- Clone the `hca-ebi-wrangler-central` repo if you haven’t already and install the requirements:
+- Clone the `hca-ebi-dev-team` repo if you haven’t already and install the requirements:
    ```
-   git clone https://github.com/ebi-ait/hca-ebi-wrangler-central.git
-   cd hca-ebi-wrangler-central/src/
+   git clone https://github.com/ebi-ait/hca-ebi-dev-team.git
+   cd scripts/populate-ingest/
    pip install -r requirements.txt
    ```
 
 #### nxn.se single cell database
 
-1) Go to the `src/` folder inside your cloned version of the `hca-ebi-wrangler-central` repository
+1) Go to the `scripts/populate-ingest/` folder inside your cloned version of the `hca-ebi-dev-team` repository
```

Comment:
Is this going to be manually run? Or will we connect it to some scheduler to be run periodically?

Reply:
manually, for now